test/topology_custom/test_tablets_removenode.py

```diff
@@ -29,6 +29,23 @@ async def run_async_cl_all(cql, query: str):
     return await cql.run_async(stmt)
 
 
+@pytest.mark.asyncio
+async def test_removenode_with_coordinator_restart(manager: ManagerClient):
+    logger.info("Bootstrapping cluster")
+    cmdline = ['--logger-log-level', 'load_balancer=debug']
+
+    servers = await manager.servers_add(3, cmdline=cmdline)
+    cql = manager.get_cql()
+
+    ks1 = await create_keyspace(cql, 3, rf=1)
+    await cql.run_async(f"CREATE TABLE {ks1}.test (pk int PRIMARY KEY, c int);")
+
+    logger.info('Removing a node')
+    await manager.server_stop(servers[2].server_id)
+    await manager.server_restart(servers[0].server_id) # topo coordinator
```

Comment:
Is it guaranteed that the topology coordinator will run on servers[0]?
Is it because it bootstrapped first?
Maybe it'd be more robust to look for it dynamically.

Reply:
Will change to discover dynamically.

Reply:
> Is it guaranteed that the topology coordinator will run on servers[0]? Is it because it bootstrapped first? 

Yes, and it will step down if quorum is lost. That shouldn't happen here, unless nodes are not responding for some reason (e.g. reactor stall). Better to be robust against this, yes.